### ISSUE TITLE
ismartgate integration configuration

### ISSUE THREAD

**Issue by siobhanellis** [home-assistant/home-assistant.github.io#27764]
### Feedback

The default user is admin, but you should never use this for obvious reasons. The best method is to define a new user for home assistant and use that account for the integration.

Also, make sure that remote access is switched on in the app.

### URL

https://www.home-assistant.io/integrations/gogogate2/

### Version

2023.6.1

**Comment by vangorra** [comment]
There's a few reasons this wasn't implemented:
- Least privilege users with gogogate2 devices produces unreliable API access.
- Least privilege users still have access to open/close the device which is what one would like to protect. Full admin access isn't any more valuable as the thing you are protecting is already accessible. 
- The devices have a history of being fairly easy to break into.
- All communication is insecure, easily sniffed, and decrypted. 
- Using support for another user adds extra setup steps and support requests.

Basically, using a least privilege user with HA and gogogate2 devices doesn't' add security value. If security is a high priority, then the best option is to run these devices on an isolated IOT network using VLANs.

**Comment by siobhanellis** [comment]
I couldn’t get it to work with the admin user at all. As soon as I created another user, it worked.

Siobhan

IOT: Practical HomeKit: ( https://practicalhomekit.blogspot.com )
Charity Fundraising: Lammie Drive: ( lammiedrive.blogspot.com<http://lammiedrive.blogspot.com/> )
